orchestration-stub/src/index/index.ts

```diff
@@ -172,10 +172,15 @@ const post = async (
   event: APIGatewayProxyEvent,
 ): Promise<APIGatewayProxyResult> => {
   const form = parseRequestParameters(event.body);
+  const previousSessionId = getCookie(event.headers["cookie"], "gs")?.split(
+    ".",
+  )[0];
   const gsCookie = await setUpSession(event.headers, form);
+
+  //Isn't the journeyId the at index [1]?
```

Comment:
Agreed, do we know what the implications of this are? (I know separate to this PR)